cloud/scope/clients.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2023 Akamai Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scope
+
+import (
+	"github.com/linode/linodego"
+)
+
+type LinodeClients struct {
```

Comment:
i might be missing context on other usages of the linodego client... is this plural on purpose? At present I don't see the benefit over doing it this way than just adding a `lindoeClient` next to the k8s client on the on the scope.

Reply:
I stole the pattern from the achieved provider and other provider and but later realized we just have one client. Thanks for the comment!

Reply:
After digging into the cluster API provider implementation, now I think the `LinodeClients` struct is still useful for centralizing Linode client lifecycle management logic, e.g. client creation. 

[This was found in the achieved version of the CAPI provider.](https://github.com/linode/cluster-api-provider-linode-archive/blob/master/cloud/scope/sessions.go#L30-L45) But yea, it is not that necessary and we should be fine for now with the a private function to create the client in cluster scope package. We can always move to this pattern later when we find it's more useful/necessary (unless you think we should do it now, lol).